physics/docs/pdftxt/RE300/FV3_HRRR_input.nml

```diff
@@ -0,0 +1,295 @@
+    knob_ugwp_ndx4lh = 1
+    knob_ugwp_solver = 2
+    knob_ugwp_source = 1, 1, 0, 0
+    knob_ugwp_stoch = 0, 0, 0, 0
+    knob_ugwp_version = 0
+    knob_ugwp_wvspec = 1, 25, 25, 25
+    launch_level = 25
+/
+
+&diag_manager_nml
+    max_output_fields = 450
+    prepend_date = .false.
+/
+
+&external_ic_nml
+    checker_tr = .false.
+    filtered_terrain = .true.
+    gfs_dwinds = .true.
+    levp = 65
+    nt_checker = 0
+/
+
+&fms_io_nml
+    checksum_required = .false.
+    max_files_r = 100
+    max_files_w = 100
+/
+
+&fms_nml
+    clock_grain = 'ROUTINE'
+    domains_stack_size = 12000000
+    print_memory_usage = .false.
+/
+
+&fv_core_nml
+    a_imp = 1.0
+    adjust_dry_mass = .false.
+    bc_update_interval = 6
+    beta = 0.0
+    consv_am = .false.
+    consv_te = 0.0
+    d2_bg = 0.0
+    d2_bg_k1 = 0.2
+    d2_bg_k2 = 0.04
+    d4_bg = 0.12
+    d_con = 1.0
+    d_ext = 0.0
+    dddmp = 0.1
+    delt_max = 0.008
+    dnats = 0
+    do_sat_adj = .false.
+    do_schmidt = .true.
+    do_vort_damp = .true.
+    dwind_2d = .false.
+    dz_min = 2
+    external_eta = .true.
+    external_ic = .true.
+    fill = .true.
+    full_zs_filter = .false.
+    fv_debug = .false.
+    fv_sg_adj = 300
+    gfs_phil = .false.
+    hord_dp = -5
+    hord_mt = 5
+    hord_tm = 5
+    hord_tr = 10
+    hord_vt = 5
+    hydrostatic = .false.
+    io_layout = 1, 1
+    k_split = 2
+    ke_bg = 0.0
+    kord_mt = 9
+    kord_tm = -9
+    kord_tr = 9
+    kord_wz = 9
+    layout = 5, 2
+    make_nh = .true.
+    mountain = .false.
+    n_split = 5
+    n_sponge = 24
+    n_zs_filter = 0
+    na_init = 1
+    ncep_ic = .false.
+    nggps_ic = .true.
+    no_dycore = .false.
+    nord = 3
+    nord_tr = 2
+    npx = 220
+    npy = 132
+    npz = 64
+    nrows_blend = 10
+    ntiles = 1
+    nudge_qv = .false.
+    nwat = 6
+    p_fac = 0.1
+    phys_hydrostatic = .false.
+    print_freq = 6
+    psm_bc = 1
+    range_warn = .true.
+    read_increment = .false.
+    regional = .true.
+    regional_bcs_from_gsi = .false.
+    res_latlon_dynamics = 'fv3_increment.nc'
+    reset_eta = .false.
+    rf_cutoff = 2000.0
+    stretch_fac = 0.999
+    target_lat = 38.5
+    target_lon = -97.5
+    tau = 5.0
+    use_hydro_pressure = .false.
+    vtdm4 = 0.02
+    warm_start = .false.
+    write_restart_with_bcs = .false.
+    z_tracer = .true.
+/
+
+&fv_grid_nml
+    grid_file = 'INPUT/grid_spec.nc'
+/
+
+!>[GFS_PHYSICS_NML]
+&gfs_physics_nml
+    bl_mynn_edmf       = 1
+    bl_mynn_edmf_mom   = 1
+    bl_mynn_tkeadvect  = .true.
+    cal_pre            = .false.
+    cdmbgwd            = 3.5, 1.0
+    cnvcld             = .false.
+    cnvgwd             = .false.
+    cplflx             = .false.
+    debug              = .false.
+    do_deep            = .false.
+    do_gsl_drag_ls_bl  = .true.
+    do_gsl_drag_ss     = .true.
+    do_gsl_drag_tofd   = .true.
+    do_mynnedmf        = .true.
+    do_mynnsfclay      = .true.
+    dspheat            = .true.
```

Comment:
In RRFS add after this line:    do_plumerise = .true.
    do_tofd = .false.
    do_ugwp = .false.
    do_ugwp_v0 = .false.
    do_ugwp_v0_nst_only = .false.
    do_ugwp_v0_orog_only = .false.
    drydep_opt = 1
    do_plumerise = .true.
    do_tofd = .false.
    do_ugwp = .false.
    do_ugwp_v0 = .false.
    do_ugwp_v0_nst_only = .false.
    do_ugwp_v0_orog_only = .false.
    drydep_opt = 1